project.yaml

```diff
@@ -6,7 +6,9 @@ expectations:
 actions:
 
   generate_study_population:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition
+    run: cohortextractor:latest generate_cohort --study-definition study_definition --output-format='csv.gz'
     outputs:
       highly_sensitive:
-        cohort: output/input.csv
+        cohort: output/input.csv.gz
+    ### NOTE if you plan to use Stata to analyse the main input file(s) please use --output-format='dta.gz' instead
```

Comment:
could add `, and subsequently open with gzuse in Stata (http://fmwww.bc.edu/RePEc/bocode/g/gzsave.html)`?

Reply:
Implemented this but didn't work in tests so reverted back to CSV option for Stata